.github/workflows/connect-publish.yml

```diff
@@ -0,0 +1,31 @@
+# This action publishes to connect using the git-backed workflow. Note that
+# it is only necessary to do it this way if the product is in a private or
+# internal repository; otherwise, it is far easier to deploy following the
+# instructions in https://docs.posit.co/connect/user/git-backed/
+
+# To use this workflow, the user will need to add 2 secrets, one named
```

Comment:
Suggestion: maybe mention manifest.json here as well and make it consistent with the instructions from R/use_r_workflows.R.